components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/launcher/launcher.js

```diff
@@ -1,9 +1,9 @@
 /**
  * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org)  Apache License, Version 2.0  http://www.apache.org/licenses/LICENSE-2.0
  */
-define(['log', 'jquery', 'backbone', 'lodash', 'context_menu', 'launch_manager', 'alerts'],
+define(['log', 'jquery', 'backbone', 'lodash', 'context_menu', 'launch_manager', 'alerts', 'utils', 'ace/ace'],
```

Comment:
Are we using ace ?

Reply:
Removed in https://github.com/wso2/carbon-analytics/pull/1870/commits/8bae65e28e84823f7dc4ee52fcae9df8e8fd8250